libphx/src/RigidBody.cpp

```diff
@@ -202,15 +202,15 @@ void RigidBody_Attach (RigidBody* parent, RigidBody* child, Vec3f* pos, Quat* ro
 
   //Convert parent to a compound
   if (!RigidBody_IsParent(parent)) {
-    btRigidBody*    pBody  = parent->handle;
```

Comment:
Crash bugfix. `CollisionShape_GetCached` returns a pointer to a dynamic array that gets resized by `CollisionShape_Create`